### PR TITLE
Add in prompt to remind students to fill in faded blocks

### PR DESCRIPTION
**Pull request by f-fromont**
image

### CODE REVIEW

File: parsons-tool-frontend/src/pages/ProblemEvaluation.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default function ProblemEvaluation() {
   const { nextProblem, currentProblem } = useProblems();
 
   const [problem, setProblem] = useState(null);
+  const [isFaded, setIsFaded] = useState(false);
```
Comment (HiruNya) [inline parsons-tool-frontend/src/pages/ProblemEvaluation.js:18]: You could also probably just do something like
```
const isFaded = curentProblem.problem.blocks.some(block => block && block.fadedIndices.length > 0);
```

**Review by HiruNya — DISMISSED** [review]
LGTM - one suggestion but minor and not important

File: parsons-tool-frontend/src/pages/ProblemEvaluation.js (hunk 1 above)
Comment (f-fromont) [reply parsons-tool-frontend/src/pages/ProblemEvaluation.js:18]: sure will try it out

**Review by HiruNya — APPROVED** [review]
LGTM

**Pull request merged**